kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -69,26 +69,19 @@ class DerivedResourcesTest {
 
     public static final Clock TEST_CLOCK = Clock.fixed(Instant.EPOCH, ZoneId.of("Z"));
 
-    public static KafkaProxy kafkaProxyFromFile(Path path) {
```

Comment:
nice, there's also a KafkaProtocolFilter load that can use `resourcesFromFiles`. Have refactored here https://github.com/k-wall/kroxylicious/pull/58

also, this moves the `assertMinimalMetadata` into `resourcesFromFiles` as all resources loaded should have a sanity check that they have name and namespace set.